osu.Game/Graphics/Containers/LinkFlowContainer.cs

```diff
@@ -70,7 +70,7 @@ public void AddLink(LocalisableString text, LinkAction action, string argument,
             createLink(spriteText.Yield(), new LinkDetails(action, argument), tooltipText);
         }
 
-        public void AddLink(IEnumerable<SpriteText> text, LinkAction action = LinkAction.External, string linkArgument = null, string tooltipText = null)
+        public void AddLink(IEnumerable<SpriteText> text, LinkAction action, string linkArgument, string tooltipText = null)
```

Comment:
Not 100% sure what the removal of the default on `LinkAction` is aiming to achieve, but it does look to be more consistent with the other methods signature-wise.

Reply:
The main change here was removing the `null` portion of `linkArgument`. There's only one usage of this function though, and it felt like explicitly requiring the action type is better.